test/appsignal/transaction/filter_test.exs

```diff
@@ -48,6 +48,12 @@ defmodule Appsignal.Transaction.FilterTest do
              ]) == %{"foo" => "bar", "password" => "[FILTERED]"}
     end
 
+    test "filter_values with an atom key" do
+      assert MapFilter.filter_values(%{"foo" => "bar", "password" => "should_not_show"}, [
```

Comment:
you're not actually testing with atoms here?

Reply:
Updated in https://github.com/appsignal/appsignal-elixir/pull/475/commits/fe5e0c37b2da53963586f1b38fb0371ef1fcce4f, this went wrong while pushing the changes.